virttest/env_process.py

```diff
@@ -805,7 +805,7 @@ def preprocess(test, params, env):
                                                 image_name_only)
 
     # firewall blocks dhcp from guest through virbr0
-    if params.get('firewalld_service', "yes") == "yes":
+    if params.get('firewalld_dhcp_workaround', "no") == "yes":
```

Comment:
@ldoktor Thank you for your time, this is the better way to implement this firewalld dhcp workaround.

As we are working to abstract the firewall configuration across avocado-vt, we still need to have `firewalld_service` param as suggested by @luckyh as `yes`, `no` and `default` (to proceed with whatever state it is in), so I have integrated your fix in PR https://github.com/avocado-framework/avocado-vt/pull/1852, please help to give your thoughts. I have included your commit and signature of your fix as well.